test/misc/update-merkle-distributor-rewards-parameters.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from 'chai';
+import { BigNumber } from 'ethers';
+
+import { describeContract, TestContext } from '../helpers/describe-contract';
+
+// LP rewards should stay the same (1150685000000000000000000)
+// Trader rewards should be reduced by 25% (3835616000000000000000000 => 2876712000000000000000000)
+// Alpha parameter should be set to 0
+
+function init() { }
+
+describeContract('', init, (ctx: TestContext) => {
```

Comment:
Suggestion: Add a description here since [other tests have it](https://github.com/dydxfoundation/governance-contracts/blob/14bea3c0c4cbeb8b1a51da43aacc303da4567836/test/tx-builder/dydx-governance.spec.ts#L29).